### DISCUSSION TITLE
建议goctl支持在生成项目的时候生成makefile文件

### DISCUSSION THREAD

**Discussion by feixiao** [zeromicro/go-zero#847] [Ideas]
makefile文件内容大致如下

build:api
	@go build -o cmd

run:
	@go run gateway.go -f etc/gateway-api.yaml

api:clean_api
	@goctl api go -api gateway.api -dir .

mysql:clean_model
	@goctl model mysql ddl -src gateway.sql -dir ./model

model:clean_model
	@goctl model mysql ddl -c -src gateway.sql -dir ./model

clean_api:
	@echo "doing clean_api"
	@-rm -rf ./internal/handler/*

clean_model:
	@-rm -rf ./model/*

**Comment by kevwan** [comment]
可以通过插件机制来实现
